<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectPathsWidget</class>
 <widget class="QWidget" name="ProjectPathsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>627</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="KComboBox" name="projectPaths">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KPushButton" name="addPath">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add a new sub-project path entry.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Add</string>
          </property>
          <property name="icon">
           <iconset theme="list-add"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KPushButton" name="replacePath">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&amp;lt;html&amp;gt;&amp;lt;head/&amp;gt;&amp;lt;body&amp;gt;&amp;lt;p&amp;gt;Replace the currently selected path with a new value keeping the includes and defines currently set for the selected path.&amp;lt;/p&amp;gt;&amp;lt;/body&amp;gt;&amp;lt;/html&amp;gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Replace</string>
          </property>
          <property name="icon">
           <iconset theme="document-properties"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KPushButton" name="removePath">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Delete current sub-project path entry.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Remove</string>
          </property>
          <property name="icon">
           <iconset theme="list-remove"/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="8" column="0">
       <widget class="KTabWidget" name="languageParameters">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>8</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="includePathsPage">
         <attribute name="title">
          <string>Includes/Imports</string>
         </attribute>
         <layout class="QVBoxLayout" name="includesLayout">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="IncludesWidget" name="includesWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>8</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="definesPage">
         <attribute name="title">
          <string>Defines</string>
         </attribute>
         <layout class="QVBoxLayout" name="gridLayout_2">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="DefinesWidget" name="definesWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>8</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>DefinesWidget</class>
   <extends>QWidget</extends>
   <header>defineswidget.h</header>
  </customwidget>
  <customwidget>
   <class>IncludesWidget</class>
   <extends>QWidget</extends>
   <header>includeswidget.h</header>
  </customwidget>
  <customwidget>
   <class>KTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
 </connections>
</ui>
